app/views/issues/show.html.slim

```diff
@@ -17,9 +17,19 @@
           .follow.pull-right
             - if user_signed_in? && current_user.active?
               = render 'shared/follow_link', issue: @issue
-            .followers-count
-              span Спостерігачів:
-              span.badge.pull-right #{@issue.followers_count}
+              .followers-count
+                span Спостерігачів:
+                span.badge.pull-right #{@issue.followers_count}
+        .votes-wrapper
+          .votes.pull-right
+            - if user_signed_in? && current_user.active?
+              - if @voted.empty?
```

Comment:
- if user_signed_in? && current_user.active?

this is repeated twice - before following block and before voting, maybe we can refactor to use this "if" just once for both follow and vote?